DataFormats/interface/DressedGenParticle.h

```diff
@@ -0,0 +1,37 @@
+#ifndef DressedGenParticle_h
+#define DressedGenParticle_h
+
+#include "UWVV/DataFormats/interface/DressedGenParticleFwd.h"
+#include "DataFormats/HepMCCandidate/interface/GenParticle.h"
+
+class DressedGenParticle : public reco::GenParticle {
+    public:
+        DressedGenParticle() {}
+        virtual ~DressedGenParticle();
+        DressedGenParticle(const LeafCandidate & c) : 
+            reco::GenParticle(c) { }
+        DressedGenParticle(const reco::GenParticle & c) : 
+            reco::GenParticle(c) { }
+        DressedGenParticle(const reco::GenParticle & cand, 
+            //const reco::GenParticleRefVector associates, float dRmax);
+            const reco::GenParticleCollection associates, float dRmax);
```

Comment:
The passed collections probably aren't big enough for it to matter, but just in case could we pass in the possible associates by reference?